tests/onnx/requirements.txt

```diff
@@ -1,7 +1,7 @@
 pytest-mock>=3.3.1
 pytest-dependency>=0.5.1
 pytest-ordering>=0.6
-torch==1.13.0
-torchvision==0.14.0
+torch~=1.13.0
```

Comment:
@vshampor why did you update torch version in tests? What if any bugs will be added in a new release version of torch? Moreover if they update the models inside torchvision?
I would prefer to keep it strict (==)

Reply:
Patch version updates tend to fix bugs, not add them.

Reply:
Ok, since the 1.13 has the security issues let's update it.